src/biome/text/helpers.py

```diff
@@ -231,3 +232,30 @@ def stringify(value: Any) -> Any:
     if isinstance(value, Iterable):
         return [stringify(v) for v in value]
     return str(value)
+
+
+def sanitize_for_params(x: Any) -> Any:
```

Comment:
Could give a more general name and change uses of `allennlp.common.util.sanitize` by this new one? 
There are some code places where we're making data sanitize (maybe in another PR)

Reply:
The purpose of the two methods is not exactly the same (one is to sanitize for serialization, this one is to sanitize for a more flexible from_params usage), but maybe they are close enough to use just one version. If you dont mind i would check this in another PR.